Sources/PopOverViewController.swift

```diff
@@ -4,19 +4,60 @@
 
 import Foundation
 
-open class PopOverViewController: UITableViewController, UIAdaptivePresentationControllerDelegate {
+open class PopOverViewController: UITableViewController, UIAdaptivePresentationControllerDelegate, UIPopoverPresentationControllerDelegate {
+    
+    fileprivate var _cornerRadius: CGFloat = 16
+    open var cornerRadius: CGFloat {
+        set {
+            _cornerRadius = newValue
+            view.superview?.layer.cornerRadius = newValue
+        }
+        get {
+            return _cornerRadius
+        }
+    }
+    
+    fileprivate var _width: CGFloat = 230
```

Comment:
This can be reduced by using a property observer e.g. didSet. So from this
```
fileprivate var _width: CGFloat = 230
open var width: CGFloat {
    set {
            _width = newValue
            view.frame.size.width = newValue
            preferredContentSize.width = newValue
        }
    get {
        return _width
    }
}
```
to
```
open var width: CGFloat = 230 {
    didSet {
        view.frame.size.width = width
        preferredContentSize.width = width
    }
}
```